.github/workflows/lint-md.yml

```diff
@@ -0,0 +1,11 @@
+name: Check Markdown links
+on:
+  pull_request:
+    paths: "**.md"
+jobs:
+  lint-md:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm install
```

Comment:
What does this install and why is it needed?

Reply:
Installs the usual package dependencies, to which this change adds `remark-cli` and `remark-validate-links` https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fe72bd3fdc8af7c6482de1819497913ba0ee1a04/package.json#L33-L34

Reply:
I'm blind.  Thanks!